### DISCUSSION TITLE
QR pole "sprava pre prijimatela"

### DISCUSSION THREAD

**Discussion by fafayqa** [xseman/bysquare#30] [General]
Vas kod dosadza do tohoto pola slovo "Platba", ale nikde v celom kode ci uz index.html alebo index.js som to nenasiel - odkial sa to tam vzalo a ako dosadit aj polia, ktore ako moznost neuvadzate vo vasom priklade pre web/lit?

**Comment by xseman** [comment]
Nerozumiem, vieš mi prosím dať príklad so so vstupnými dátami a očakávaným výstupom?
